site/assets/scss/_navbar.scss

```diff
@@ -51,8 +51,23 @@
     @include media-breakpoint-down(lg) {
       border-top: 1px solid #666;
     }
+
+    .dropdown-toggle {
+      line-height: 1.5625rem;
+
+      @include media-breakpoint-up(md) {
+        line-height: 1.875rem;
+      }
+    }
   }
   // End mod
 }
 
+// Boosted mod
+.main-header {
```

Comment:
The dropdown menu is not visible in desktop mode, how about we change it to the following instead? Does it fit what you had in mind?

```scss
@include media-breakpoint-down(lg) {
  .main-header {
    max-height: 100vh;
    overflow: hidden auto;
  }
}
```